packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -241,6 +239,67 @@ class ChromiumLauncher {
     ), skipCheck);
   }
 
+  Future<Process> _spawnChromiumProcess(List<String> args) async {
+    // Keep attempting to launch the browser until one of:
+    // - Chrome launched successfully, in which case we just return from the loop.
+    // - The tool detected an unretriable Chrome error, in which case we throw ToolExit.
+    while (true) {
+      final Process process = await _processManager.start(args);
+
+      bool exited = false;
+      unawaited(process.exitCode.then((int exitCode) {
```

Comment:
since you don't use the exitCode, use `whenComplete(() { ... })`

Reply:
Done.